packages/@sanity/schema/src/sanity/validation/types/deprecated.ts

```diff
@@ -0,0 +1,43 @@
+import {SchemaValidationResult} from '../../typedefs'
+import {warning} from '../createValidationResult'
+
+export function validateDeprecatedProperties(type): SchemaValidationResult[] {
+  const warnings = []
+
+  // Do not validate document types
+  if (type?.type === 'document') return []
```

Comment:
Would rather see a comment explaining _why_ we dont validate document types. Cause I was like, why not, here?

Reply:
Good point